src/SamplesApp/SamplesApp.UITests/Microsoft_UI_Xaml_Controls/NumberBoxTests/Given_NumberBox.cs

```diff
@@ -175,92 +176,15 @@ public void UpDownEnabledTest()
 		[AutoRetry]
 		public void BasicExpressionTest()
 		{
-			Run("UITests.Shared.Microsoft_UI_Xaml_Controls.NumberBoxTests.MUX_Test");
+			Run("UITests.Microsoft_UI_Xaml_Controls.NumberBoxTests.NumberBox_ExpressionTest");
+
+			_app.Marked("TestNumberBox").WaitUntilExists();
+
+			_app.Marked("RunButton").FastTap();
+
+			_app.WaitFor(()=> _app.Marked("Status").GetText() is string s && (s.Equals("Success") || s.StartsWith("Failure")));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this call to 's.StartsWith' to an overload that accepts a 'StringComparison' as a parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5037645)